bin/anthology/bibtex.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import re
+import logging
+import pybtex, pybtex.database.input.bibtex
+
+
+def read_bibtex(bibfilename):
+    # Guess encoding. BibTeX is theoretically always in ASCII
```

Comment:
Maybe a one-line docstring for the method?  "Reads bibtex data from a file into pybtex data structures, guessing the encoding of the file" or something similar.